s5/conditional.rb

```diff
@@ -0,0 +1,64 @@
+# if
+word = "Udemy"
+if word.include?("my")
+  puts "Yes, #{word} includes \"my\""
+end
+
+# truthiness n falsiness
+if !nil
+  puts "nil is falsey"
+end
+
+if (4 && [] && {} && 4.6)
+  puts "All this statements are truthy"
+end
+
+# Strings T/F
+s = ""
+# if "" && "Hola"
+if s
+  puts "  Strings are truthy,
+  but it send out an alert
+  if you put them directly."
+end
+puts
+
+# elsif
+if nil
+  puts "This will never print"
+elsif !nil
```

Comment:
You can skip this validation. Just use `else`.

Reply:
Yes. I took it into account when I typed it, I did it more to emphasize when I reread it that you can deny a _null_.